maat-court-data-api/src/main/java/gov/uk/courtdata/dces/service/DebtCollectionService.java

```diff
@@ -49,16 +50,12 @@ public boolean updateContributionFileReceivedCount(Integer fileId) {
             log.info("No associated file was found for contribution");
             return false;
         }
-        ContributionFilesEntity filesEntity = getContributionFile(fileId);
-        if (Objects.nonNull(filesEntity)) {
-            filesEntity.incrementReceivedCount();
-            filesEntity.setDateReceived(LocalDate.now());
-            debtCollectionRepository.updateContributionFilesEntity(filesEntity);
-            log.info("update contribution_file ID {}: completed successfully", fileId);
-            return true;
-        } else {
-            throw new MAATCourtDataException("update contribution_file ID " + fileId + ": not found");
+        int rowsUpdated = contributionFilesRepository.incrementRecordsReceived(fileId, USER_AUDIT);
+        log.info("update contribution_file ID {} records received: {} row(s) updated", fileId, rowsUpdated);
+        if (rowsUpdated != 1) {
+            throw new MAATCourtDataException("update contribution_file ID " + fileId + " records received: " + rowsUpdated + " row(s) updated");
```

Comment:
I would consider using a String format here instead of concatenation to aid readability, but it is pretty minor point and more personal taste TBH.